corehq/form_processor/migrations/0092_auto_20200924_1753.py

```diff
@@ -14,8 +14,8 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.RunSQL(
-            f"ALTER INDEX {OLD_NAME} RENAME TO {NEW_NAME}",
-            f"ALTER INDEX {NEW_NAME} RENAME TO {OLD_NAME}",
+            f"ALTER INDEX IF EXISTS {OLD_NAME} RENAME TO {NEW_NAME}",
```

Comment:
I know this was my suggestion, but reading this migration more, can we just eliminate these two lines? Together they net to doing nothing, and seem to contradict the `state_operations`. They leave the index named `OLD_NAME`, but the state operation says its adding one with name `NEW_NAME`, which should mess up its internal understanding of db state. 

This change does handle the case that someone already has an index `NEW_NAME`, and will successfully change it to `OLD_NAME`, which is I think the situation Clayton and Steph were in, although this would end up in a different result than they did by faking this.

Reply:
Added 1af2f649d34 but will wait for a comment from @snopoke since he originally wrote that code.